packages/react-heartwood-components/src/components/RecordSelectionList/RecordSelectionList.js

```diff
@@ -130,6 +130,23 @@ export default class RecordSelectionList extends Component<
 		}
 	}
 
+	async reset() {
+		const { loadRecords } = this.props
+
+		const initialRecords = await loadRecords({
+			offset: 0,
```

Comment:
Should these values come from `defaultProps`?

Reply:
Yeah good call; I went ahead and did this across the file. Also inverted the default for another default prop, since that feature should be opt-in.